### PR TITLE
Fixes #26481 - Add foreman.service systemd file

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Related to discussions started here -- https://github.com/theforeman/foreman-packaging/pull/3315#issuecomment-477594706

File: extras/systemd/foreman.service [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Documentation=https://theforeman.org
+After=network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=simple
+User=$FOREMAN_USER
+TimeoutSec=300
+WorkingDirectory=/usr/share/foreman
+ExecStart=/usr/bin/scl enable tfm "rails server -e $FOREMAN_ENV -p $FOREMAN_PORT -b $FOREMAN_BIND"
+Environment=FOREMAN_ENV=production FOREMAN_PORT=3000 FOREMAN_BIND=0.0.0.0 FOREMAN_USER=foreman
```
Comment (timogoebel) [inline extras/systemd/foreman.service:12]: Should this be `RAILS_ENV`?

File: extras/systemd/foreman.service [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+EnvironmentFile=-/etc/sysconfig/foreman
```
Comment (ekohl) [inline extras/systemd/foreman.service]: This is a RH thing to do. Debian has `/etc/default`. IIRC systemd recommended using drop in files instead. In foreman-proxy we left it out and then added it back in packaging. IMHO that's still the best thing to do.

File: extras/systemd/foreman.service (hunk 1 above)
Comment (ekohl) [reply extras/systemd/foreman.service:12]: It's passed in via `-e` so I think it's not needed

File: extras/systemd/foreman.service [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+Description=Foreman
+Documentation=https://theforeman.org
+After=network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=simple
+User=$FOREMAN_USER
+TimeoutSec=300
+WorkingDirectory=/usr/share/foreman
+ExecStart=/usr/bin/scl enable tfm "rails server -e $FOREMAN_ENV -p $FOREMAN_PORT -b $FOREMAN_BIND"
```
Comment (ekohl) [inline extras/systemd/foreman.service]: Are there long options we could use?

Comment (ehelms) [reply extras/systemd/foreman.service]: Updated

File: extras/systemd/foreman.service (hunk 2 above)
Comment (ehelms) [reply extras/systemd/foreman.service]: Dropped this line

**Comment by ehelms** [comment]
Updated based on comments

**Review by ekohl — APPROVED** [review]

**Comment by ehelms** [comment]
[test foreman]

**Comment by ehelms** [comment]
[test upgrade]

**Comment by mmoll** [comment]
merged, thanks @ehelms!

**Pull request merged**